pkg/cloudprovider/controller/nodecontroller.go

```diff
@@ -326,6 +331,14 @@ func (s *NodeController) checkNodeReady(node *api.Node) *api.NodeCondition {
 	}
 }
 
+// sendConditionEvent sends condition change event.
+func (s *NodeController) sendConditionEvent(node *api.Node, kind api.NodeConditionKind, oldCondition, newCondition *api.NodeCondition) {
+	// TODO: if 'oldCondition' is nil, we should publish the initial condition, but selfLink is not set.
+	if oldCondition != nil {
```

Comment:
Doh, we can use one that's returned by Create() methon.  I'll take another look.